API/Entities/MangaFile.cs

```diff
@@ -21,6 +21,10 @@ public class MangaFile : IEntityDate
     /// </summary>
     public required string FilePath { get; set; }
     /// <summary>
+    /// A hash of the document using Koreader's unique hashing algorithm
+    /// </summary>
+    public string KoreaderHash { get; set; }
```

Comment:
This should be `string?` as we aren't using for all file types. I would also update `<remarks>` to include information about the supported file types.